src/wallet/index.ts

```diff
@@ -72,13 +86,14 @@ class Wallet {
     return Wallet.deriveWallet(seed, algorithm)
   }
 
-  /**
-   * Derives a wallet from a secret (AKA a seed)
-   *
-   * @param secret - A string used to generate a keypair (publicKey/privateKey) to derive a wallet.
-   * @param algorithm - The digital signature algorithm to generate an address fro.
-   * @returns A Wallet derived from a secret (AKA a seed).
-   */
+  // .
```

Comment:
This should be changed to the `/**` syntax used above (you just need to add a period to the end of the line.